### PR TITLE
Refresh the INFO page to add missing fields

### PR DESCRIPTION
**Pull request by enjoy-binbin**
The INFO page kind of outupdates over time, this PR
reflashes it based on the latest unstable branch.

### CODE REVIEW

File: docs/info-sections.md [hunk 1]
```diff
@@ -94,23 +105,63 @@ used_disk_percent: 53%
 estimate_keys[default]:0
 block_cache_usage[default]:0
 block_cache_pinned_usage[default]:0
-index_and_filter_cache_usage:[default]:0
```
Comment (enjoy-binbin) [inline docs/info-sections.md:97]: the diff is:
```diff
- index_and_filter_cache_usage:[default]:0
+ index_and_filter_cache_usage:[default]:0
```

Comment (enjoy-binbin) [reply docs/info-sections.md:97]: everything else is new

**Comment by enjoy-binbin** [comment]
for reviewer, i suggest for now we drop the whitespace diff:
image

**Comment by enjoy-binbin** [comment]
There are some fields I'm not sure about, I left blank as xxx.
Please help to insert suggestions, or update them with a new PR

**Comment by git-hulk** [comment]
> There are some fields I'm not sure about, I left blank as xxx. Please help to insert suggestions, or update them with a new PR

Thanks @enjoy-binbin, let me fill this later

File: docs/info-sections.md [hunk 2]
```diff
@@ -272,18 +340,25 @@ There are five column families on kvrocks:
...
+|-----------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| estimate_keys[xxx]                      | Estimate keys in the column family, may contains the tombstone and expired keys, it's a fast way to know how many keys on the column family but not precise. |
+| block_cache_usage[xxx]                  | Total block cache bytes used by this column family.                                                                                                          |
+| block_cache_pinned_usage[xxx]           | Total pinned bytes in this column family.                                                                                                                    |
+| index_and_filter_cache_usage[xxx]       | Total bytes was used to cache the index and filter block.                                                                                                    |
+| level0_file_limit_slowdown[xxx]         | xxx.                                                                                                    |
```
Comment (git-hulk) [inline docs/info-sections.md:356]:
```suggestion
| level0_file_limit_slowdown[xxx]         | Number of IO write stalls caused by reaching the level0 file slowdown limit.                                                                                                    |
```

File: docs/info-sections.md [hunk 3]
```diff
@@ -272,18 +340,25 @@ There are five column families on kvrocks:
...
+| level0_file_limit_stop[xxx]             | xxx.                                                                                                    |
+| pending_compaction_bytes_slowdown[xxx]  | xxx.                                                                                                    |
+| pending_compaction_bytes_stop[xxx]      | xxx.                                                                                                    |
+| memtable_count_limit_slowdown[xxx]      | xxx.                                                                                                    |
```
Comment (git-hulk) [inline docs/info-sections.md:360]:
```suggestion
| memtable_count_limit_slowdown[xxx]      | Number of IO write stalls caused by reaching the memtable count slowdown limit.                                                                                              |
```

File: docs/info-sections.md [hunk 4]
```diff
@@ -272,18 +340,25 @@ There are five column families on kvrocks:
...
+| memtable_count_limit_stop[xxx]          | xxx.                                                                                                    |
```
Comment (git-hulk) [inline docs/info-sections.md:361]:
```suggestion
| memtable_count_limit_stop[xxx]          | Number of IO write stalls caused by reaching the memtable count stop limit.                                                                                                  |
```

File: docs/info-sections.md [hunk 5]
```diff
@@ -298,8 +373,14 @@ There are five column families on kvrocks:
 | compaction_pending      | This metric returns 1 if at least one compaction is pending; otherwise, the metric reports 0.                                              |
 | num_running_compactions | Number of currently running compactions.                                                                                                   |
 | num_live_versions       | Number of live versions. More live versions often mean more SST files are held from being deleted, by iterators or unfinished compactions. |
+| num_super_version       | xxx. |
```
Comment (git-hulk) [inline docs/info-sections.md:376]:
```suggestion
| num_super_version       | Number of the super version inside the RocksDB. |
```

File: docs/info-sections.md [hunk 6]
```diff
@@ -272,18 +340,25 @@ There are five column families on kvrocks:
 |---------------|------------------------------------------------------------------------------------------------------------|
 | default       | Used to store the subkeys of the complex data structure like hash/set/list/zset/geo.                       |
 | metadata      | Used to store the metadata of the complex data structure and string.                                       |
-| zset score    | Used to store the mapping of zset's score to member, which would make the range by score operation faster. |
+| zset_score    | Used to store the mapping of zset's score to member, which would make the range by score operation faster. |
 | pubsub        | Used to propagate the pubsub message to replicas.                                                          |
 | propagate     | Used to propagate other commands to replicas except pubsub message.                                        |
+| stream        | todo xxx. |
```
Comment (enjoy-binbin) [inline docs/info-sections.md:346]: the last one

**Review by git-hulk — APPROVED** [review]

File: docs/info-sections.md (hunk 6 above)
Comment (git-hulk) [inline docs/info-sections.md:346]:
```suggestion
| stream        | Used to store the data of the stream type. |
```

File: docs/info-sections.md [hunk 7]
```diff
@@ -132,26 +188,32 @@ is_compacting:no
...
-| git_sha1          | Git SHA1                                                       |
-| os                | Operating system hosting the Kvrocks server                    |
-| arch_bits         | Architecture (32 or 64 bits)                                   |
-| gcc_version       | Version of the GCC compiler used to compile the Kvrocks server |
-| process_id        | PID of the server process                                      |
-| tcp_port          | TCP/IP listen port                                             |
-| uptime_in_seconds | Number of seconds since Kvrocks server start                   |
-| uptime_in_days    | Same value expressed in days                                   |
+| Property          | Desc                                                             |
+|-------------------|----------------------------------------------------------------- |
```
Comment (enjoy-binbin) [inline docs/info-sections.md:192]:
```suggestion
|-------------------|------------------------------------------------------------------|
```

**Pull request merged**